plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -46,12 +48,16 @@ const uint16_t EuroliteProFactory::VENDOR_ID_MK2 = 0x0403;
 
 const char EuroliteProFactory::ENABLE_EUROLITE_MK2_KEY[] =
     "enable_eurolite_mk2";
+const char EuroliteProFactory::EUROLITE_SERIALS_KEY[] =
+    "eurolite_serials";
```

Comment:
At which point this would be singular:
```suggestion
const char EuroliteProFactory::EUROLITE_SERIAL_KEY[] =
    "eurolite_serial";
```

But I think should also mention the mk2:
```suggestion
const char EuroliteProFactory::EUROLITE_MK2_SERIAL_KEY[] =
    "eurolite_mk2_serial";
```